Command/Commands/GetCurrentWeatherCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using AppConfig;
+using AppConfig.Interfaces;
+using BL.Interfaces;
+using BL.Services;
+using Command.Interfaces;
+using DAL.Interfaces;
+using DAL.Repositories;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Command.Commands
+{
+    public class GetCurrentWeatherCommand : ICommand
+    {
+        private static IConfig _config = new Config();
+        private static readonly HttpClient _client = new HttpClient();
+        private readonly string _key;
+        private readonly string _currentWeatherUrl;
+        private readonly string _forecastUrl;
+        private readonly int _forecastHour;
+        private readonly string _coordinatesUrl;
+        private IWeatherService _weatherService;
+        private IWeatherRepository _weatherRepository;
+        private IValidator _validator;
+
+        public string Text => ": Get current weather";
+
+        public GetCurrentWeatherCommand()
+        {
+            _key = _config.Key;
```

Comment:
Memory is allocated for unnecessary variables. It's better to pass value to WeatherRepository via _config